### PR TITLE
Add "free template" opportunity to inbox

### PR DESCRIPTION
**Pull request by truongwp**
This fixes issue https://github.com/Strategy11/formidable-pro/issues/2885

### CODE REVIEW

**Comment by truongwp** [comment]
@stephywells I have a commitment to do the compatibility thing. So you can take a look.

File: js/formidable_admin.js [hunk 1]
```diff
@@ -6009,9 +6033,16 @@ function frmAdminBuildJS() {
 			}
 		});
 
-		urlParams = new URLSearchParams( window.location.search );
 		if ( urlParams.get( 'triggerNewFormModal' ) ) {
 			triggerNewFormModal();
+
+			if ( isShowFreeTemplatesFormFirst ) {
+				firstLockedTemplate = jQuery( 'li.frm-locked-template.frm-free-template:eq(0)' );
```
Comment (Crabcyborg) [inline js/formidable_admin.js:6040]: https://api.jquery.com/eq-selector/

`As of jQuery 3.4, the :eq pseudo-class is deprecated. Remove it from your selectors and filter the results later using .eq().`

I know I had to replace some lines like this. They suggest using `.eq` now.

File: classes/controllers/FrmInboxController.php [hunk 2]
```diff
@@ -99,4 +100,33 @@ private static function add_tracking_request() {
...
+			return;
+		}
+
+		$link = admin_url( 'admin.php?page=formidable&triggerNewFormModal=1&free-templates=1' );
+
+		$message = new FrmInbox();
+		$message->add_message(
+			array(
+				'key'     => 'free_templates',
+				'message' => 'Just add your email address and you \'ll get a code for 10+ free form templates.',
```
Comment (Crabcyborg) [inline classes/controllers/FrmInboxController.php:125]: It looks like there is a space here that shouldn't be in `you'll`. But I doubt we want to use that word anyway.

I think even just `Add your email address to get a code for 10+ free form templates.` covers this with fewer words.

File: js/formidable_admin.js [hunk 3]
```diff
@@ -5798,32 +5807,38 @@ function frmAdminBuildJS() {
 			setTemplateCount( $li.closest( '.accordion-section' ).get( 0 ) );
 		});
 
+		showFreeTemplatesForm = function( $el ) {
+			formContainer = document.getElementById( 'frmapi-email-form' );
```
Comment (Crabcyborg) [inline js/formidable_admin.js:5811]: It doesn't look like there is actually a var declaration for this after it was copied and pasted around a bit.

**Comment by Crabcyborg** [comment]
@truongwp thanks!

This looks good other than a few notes I added. I haven't tested it yet either, but I think this is probably already good to go soon.

**Comment by truongwp** [comment]
@Crabcyborg Thanks for checking this. Let me check your notes.

**Comment by truongwp** [comment]
@Crabcyborg I have fixed the. Please check when you have time.

**Review by Crabcyborg — APPROVED** [review]
I think this looks good! Thanks @truongwp.

**Comment by truongwp** [comment]
@Crabcyborg @jairoprez Thank you.

**Pull request merged**